content/source/docs/enterprise/private/diagnostics.html.md

```diff
@@ -82,10 +81,40 @@ hashicorp-support.tar.gz.enc
 
 Then, attach the bundle to your support ticket. If possible, use the SendSafely integration (as it allows for large file uploads).
 
-### Windows
+### Scrubbing Secrets
+
+If you have extremely sensitive data in your Terraform build logs you
+may opt to omit these logs from your bundle. However, this may impede our
+efforts to diagnose any problems you are encountering. To create a custom
+support bundle, run the following commands:
+
+    sudo -s
+    hashicorp-support
+    cd /var/lib/hashicorp-support
+    tar -xzf hashicorp-support.tar.gz
+    rm hashicorp-support.tar.gz*
+    rm nomad/*build-worker*
+    tar -czf hashicorp-support.tar.gz *
+    gpg2 -e -r "Terraform Enterprise Support" \
+        --cipher-algo AES256 \
+        --compress-algo ZLIB \
+        -o hashicorp-support.tar.gz.enc \
+        hashicorp-support.tar.gz
+
+You will note that we first create a support bundle using the normal procedure,
+extract it, remove the files we want to omit, and then create a new one.
+
+
+## Windows
```

Comment:
here, you dropped this: `#`

Reply:
Ah yes. This was intentional, but mistaken, since it only applies to the AMI.